### ISSUE TITLE
Played and abandoned

### ISSUE THREAD

**Issue by Mick1303** [JeffSackmann/tennis_atp#193]
As much as I would love to have all possible tennis data, I have to exclude the matches that are marked "Played and abandoned". Your format as well as mine has a winner and a loser. But in this case there is neither. 
Do you have any engine that accounts for this result (like excluding it from the queries which count win-loss records, etc.)? I decided a while ago that I will not bother with such extensions to my queries and just not include these matches. They are primarily Davis Cup and they do not affect the record of the tie. 
If you don't want to exclude them, you'll have to think about how to account for them properly to avoid misleading results.

**Comment by JeffSackmann** [comment]
yeah, it's an important thing to remember. I like to keep the results, and I need to make sure they don't show up in W-L records on the tennisabstract site. These should all have 'ABD' in the score (instead of the full "played and abandoned" which shows up on the Davis Cup and ATP website in a couple of places.
